pkg/oc/cli/cmd/export.go

```diff
@@ -66,6 +66,8 @@ func NewCmdExport(fullName string, f *clientcmd.Factory, in io.Reader, out io.Wr
 			kcmdutil.CheckErr(err)
 		},
 	}
+	cmd.Deprecated = "Use the `get --export` command instead."
```

Comment:
:tada: :tada: :tada:

Reply:
Please set `Hidden` as well, I need to swipe through all our commands which are deprecated and set that flag, so they don't appear in completion, or in help.